src/public/services/FormSgSdkService.ts

```diff
@@ -0,0 +1,16 @@
+import formSgSdk from '@opengovsg/formsg-sdk'
+import { PackageMode } from '@opengovsg/formsg-sdk/dist/types'
+
+import { TRANSACTION_EXPIRE_AFTER_SECONDS } from '../../shared/util/verification'
+
+// Reading from process.env.FORMSG_SDK_MODE defaults to undefined
+// This is because browsers are sandboxed and hence have to manually check on window...
```

Comment:
yeap, i understood the injection - the reason why `process.env` isn't relevant is because this is part of the browser and hence has no access to `process.env` (due to sandboxing) isn't it?

Reply:
I _think_ the reason why `process.env` is not relevant is that `process.env` is specific to NodeJS, and the browser is not a NodeJS process.

even if the browser process itself had a `process.env` which we can't access due to sandboxing, there's no reason why the browser process would contain our env vars, so sandboxing still doesn't come into the picture.

Reply:
actually i always thought it was cos the browser is ran in a sandbox and it doesn't have access to the `process.env` (less what the project itself defines); lemme go read up on this tomorrow and tell you what i find HAHA

Reply:
you're right! thanks for correcting my misunderstanding and on that note (node), i'll edit the comment as well!